telemetry/definitions/commonDefinitions.json

```diff
@@ -4720,6 +4725,12 @@
             "name": "codewhisperer_userModification",
             "description": "Percentage of user modifications for the selected suggestion until a fixed period of time",
             "metadata": [
+                {
+                    "type": "codewhispererCharactersAccepted"
```

Comment:
@andrewyuq fyi since there is already an existing field `codewhispererCharactersAccepted` which servers the same or similar purpose as what we proposed `codewhispererCharactersTotal`, so I choose to reuse the existing one